rulebook.tex

```diff
@@ -1364,32 +1364,41 @@ \subsubsection{Production Plan}
 requiring 1 base, and two colors requiring no additional base at
 all. Products retrieved from the Storage Station may be used only for
 $C_0$ orders with a requested quantity equal to or greater than two.
-
-In each game, up to 10 orders will be placed.\footnote{There will up
-  to 9 orders in regular game time, and an additional order in the
-  case of overtime. The number can be adjusted during the tournament
-  by the Technical Committee, should this turn out to be necessary.}
+An order can be \emph{competitive} or \emph{non-competitive}. For a
+competitive order, only the team that delivers first will score points
+for the delivery, and it will score twice the amount of points. The
+other team will not get any points for the delivery, even if it delivers
+in time.  For a non-competitive order, both teams score on delivery with
+a factor of $1$.
+
+In each game, up to 9 orders will be placed within the regular game.
+4 of those 9 orders will be \emph{competitive}, while the remaining
+orders will be \emph{non-competitive}. In case of overtime, an
+additional competitive order will be placed.\footnote{The numbers can
+be adjusted during the tournament by the Technical Committee, should
+this turn out to be necessary.}
 The complexities of orders will be similar for all games within a
-tournament phase. But the actual production chains are
-randomized. Each order will require one or more products of a
-specified product type to be delivered. The product is specified in
-terms of base color, rings (color and order), cap color, and possible
-delivery gate. The delivery time slot will have a randomized start
-time. The duration will be randomized between 30 to 180 seconds. The
-end time shall be within the game time. There is not necessarily an
-open order for a particular product at a specific time, or any order
-at all. The first order will be posted within the first 120 seconds of
-the game (but not necessarily at the beginning of the game). Orders
-for the products of complexity $C_2$ or $C_3$ will not begin in the
-first 300 seconds. An order will be announced before the delivery time
-slots starts. For $C_0$ products the lead time ahead of the start of
-the time window is 60 to 120 seconds, for $C_1$ it is 120 to 300
-seconds, and for $C_2$ and $C_3$ orders the time is between 300 and
-600 seconds. Two early orders will be posted that are announced in the
-first third of the game with a delivery time window in the last
-third. For each of the complexities $C_0$ and $C_1$ a standing order
-for a single product will be placed at the beginning of the production
-phase, which can be fulfilled at any time during the game.
+tournament phase. But the actual production chains are randomized.
```

Comment:
What has changed here? Is it just reformatting? In that case, remove. Otherwise, please give a hint.

Reply:
I added the sentence
> A competitive order always requires exactly one product to be delivered.